<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mbClientUi</class>
 <widget class="QMainWindow" name="mbClientUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1239</width>
    <height>752</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client</string>
  </property>
  <property name="windowIcon">
   <iconset resource="client_rsc.qrc">
    <normaloff>:/client/icons/client.ico</normaloff>:/client/icons/client.ico</iconset>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1239</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionFileNew"/>
    <addaction name="separator"/>
    <addaction name="actionFileOpen"/>
    <addaction name="actionFileRecent"/>
    <addaction name="actionFileClose"/>
    <addaction name="actionFileSave"/>
    <addaction name="actionFileSaveAs"/>
    <addaction name="actionFileEdit"/>
    <addaction name="actionFileImportProject"/>
    <addaction name="actionFileInfo"/>
    <addaction name="separator"/>
    <addaction name="actionFileQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionViewProject"/>
    <addaction name="actionViewLogView"/>
   </widget>
   <widget class="QMenu" name="menuDevice">
    <property name="title">
     <string>Device</string>
    </property>
    <addaction name="actionDeviceNew"/>
    <addaction name="actionDeviceEdit"/>
    <addaction name="actionDeviceDelete"/>
    <addaction name="separator"/>
    <addaction name="actionDeviceImport"/>
    <addaction name="actionDeviceExport"/>
   </widget>
   <widget class="QMenu" name="menuDataView">
    <property name="title">
     <string>Data</string>
    </property>
    <addaction name="actionDataViewItemNew"/>
    <addaction name="actionDataViewItemEdit"/>
    <addaction name="actionDataViewItemInsert"/>
    <addaction name="actionDataViewItemDelete"/>
    <addaction name="separator"/>
    <addaction name="actionDataViewImportItems"/>
    <addaction name="actionDataViewExportItems"/>
    <addaction name="separator"/>
    <addaction name="actionDataViewNew"/>
    <addaction name="actionDataViewEdit"/>
    <addaction name="actionDataViewInsert"/>
    <addaction name="actionDataViewDelete"/>
    <addaction name="separator"/>
    <addaction name="actionDataViewImport"/>
    <addaction name="actionDataViewExport"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelpAbout"/>
    <addaction name="actionHelpAboutQt"/>
    <addaction name="separator"/>
    <addaction name="actionHelpContents"/>
   </widget>
   <widget class="QMenu" name="menuRuntime">
    <property name="title">
     <string>Runtime</string>
    </property>
    <addaction name="actionRuntimeStartStop"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionEditCut"/>
    <addaction name="actionEditCopy"/>
    <addaction name="actionEditPaste"/>
    <addaction name="actionEditInsert"/>
    <addaction name="actionEditEdit"/>
    <addaction name="actionEditDelete"/>
    <addaction name="separator"/>
    <addaction name="actionEditSelectAll"/>
   </widget>
   <widget class="QMenu" name="menuPort">
    <property name="title">
     <string>Port</string>
    </property>
    <addaction name="actionPortNew"/>
    <addaction name="actionPortEdit"/>
    <addaction name="actionPortDelete"/>
    <addaction name="separator"/>
    <addaction name="actionPortNewDevice"/>
    <addaction name="actionPortClearAllDevices"/>
    <addaction name="separator"/>
    <addaction name="actionPortImport"/>
    <addaction name="actionPortExport"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <widget class="QMenu" name="menuWindowDataViews">
     <property name="title">
      <string>DataViews</string>
     </property>
     <addaction name="actionWindowDataViewCloseAll"/>
     <addaction name="separator"/>
    </widget>
    <addaction name="actionWindowViewSubWindow"/>
    <addaction name="actionWindowViewTabbed"/>
    <addaction name="separator"/>
    <addaction name="menuWindowDataViews"/>
    <addaction name="separator"/>
    <addaction name="actionWindowCloseAll"/>
    <addaction name="separator"/>
    <addaction name="actionWindowTile"/>
    <addaction name="actionWindowCascade"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionToolsSettings"/>
    <addaction name="actionToolsSendMessage"/>
    <addaction name="actionToolsSendBytes"/>
    <addaction name="actionToolsScanner"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuPort"/>
   <addaction name="menuDevice"/>
   <addaction name="menuDataView"/>
   <addaction name="menuTools"/>
   <addaction name="menuRuntime"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockLogView">
   <property name="windowTitle">
    <string>LogView</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="wgtLogView">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="leftMargin">
      <number>4</number>
     </property>
     <property name="topMargin">
      <number>4</number>
     </property>
     <property name="rightMargin">
      <number>4</number>
     </property>
     <property name="bottomMargin">
      <number>4</number>
     </property>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockProject">
   <property name="windowTitle">
    <string>Project</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockProjectContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="leftMargin">
      <number>4</number>
     </property>
     <property name="topMargin">
      <number>4</number>
     </property>
     <property name="rightMargin">
      <number>4</number>
     </property>
     <property name="bottomMargin">
      <number>4</number>
     </property>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFileNew"/>
   <addaction name="actionFileOpen"/>
   <addaction name="actionFileSave"/>
   <addaction name="separator"/>
   <addaction name="actionPortNew"/>
   <addaction name="actionDeviceNew"/>
   <addaction name="separator"/>
   <addaction name="actionDataViewNew"/>
   <addaction name="actionDataViewItemNew"/>
   <addaction name="separator"/>
   <addaction name="actionToolsSendMessage"/>
   <addaction name="actionToolsSendBytes"/>
   <addaction name="actionToolsScanner"/>
   <addaction name="separator"/>
   <addaction name="actionRuntimeStartStop"/>
  </widget>
  <action name="actionFileNew">
   <property name="icon">
    <iconset resource="../../core/gui/core_rsc.qrc">
     <normaloff>:/core/icons/filenew.png</normaloff>:/core/icons/filenew.png</iconset>
   </property>
   <property name="text">
    <string>New...</string>
   </property>
  </action>
  <action name="actionFileOpen">
   <property name="icon">
    <iconset resource="../../core/gui/core_rsc.qrc">
     <normaloff>:/core/icons/fileopen.png</normaloff>:/core/icons/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionHelpContents">
   <property name="text">
    <string>Contents</string>
   </property>
  </action>
  <action name="actionFileSave">
   <property name="icon">
    <iconset resource="../../core/gui/core_rsc.qrc">
     <normaloff>:/core/icons/filesave.png</normaloff>:/core/icons/filesave.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionFileSaveAs">
   <property name="icon">
    <iconset resource="../../core/gui/core_rsc.qrc">
     <normaloff>:/core/icons/filesaveall.png</normaloff>:/core/icons/filesaveall.png</iconset>
   </property>
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionViewProject">
   <property name="text">
    <string>Project</string>
   </property>
  </action>
  <action name="actionViewLogView">
   <property name="text">
    <string>LogView</string>
   </property>
  </action>
  <action name="actionRuntimeStartStop">
   <property name="icon">
    <iconset resource="../../core/gui/core_rsc.qrc">
     <normaloff>:/core/icons/play.png</normaloff>:/core/icons/play.png</iconset>
   </property>
   <property name="text">
    <string>Start/Stop</string>
   </property>
  </action>
  <action name="actionDataViewInsert">
   <property name="text">
    <string>Insert Data View</string>
   </property>
  </action>
  <action name="actionDataViewNew">
   <property name="icon">
    <iconset resource="../../core/gui/core_rsc.qrc">
     <normaloff>:/core/icons/dataview.png</normaloff>:/core/icons/dataview.png</iconset>
   </property>
   <property name="text">
    <string>New Data View...</string>
   </property>
  </action>
  <action name="actionDataViewDelete">
   <property name="text">
    <string>Delete Data View</string>
   </property>
  </action>
  <action name="actionDeviceNew">
   <property name="icon">
    <iconset resource="../../core/gui/core_rsc.qrc">
     <normaloff>:/core/icons/device.png</normaloff>:/core/icons/device.png</iconset>
   </property>
   <property name="text">
    <string>New Device...</string>
   </property>
  </action>
  <action name="actionDeviceDelete">
   <property name="text">
    <string>Delete Device</string>
   </property>
  </action>
  <action name="actionDeviceEdit">
   <property name="text">
    <string>Edit Device...</string>
   </property>
  </action>
  <action name="actionFileEdit">
   <property name="icon">
    <iconset resource="../../core/gui/core_rsc.qrc">
     <normaloff>:/core/icons/edit.png</normaloff>:/core/icons/edit.png</iconset>
   </property>
   <property name="text">
    <string>Edit...</string>
   </property>
  </action>
  <action name="actionFileQuit">
   <property name="icon">
    <iconset resource="../../core/gui/core_rsc.qrc">
     <normaloff>:/core/icons/quit.png</normaloff>:/core/icons/quit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionHelpAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionHelpAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionEditInsert">
   <property name="text">
    <string>Insert</string>
   </property>
  </action>
  <action name="actionEditDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionEditCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionEditCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionEditPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionEditSelectAll">
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="actionPortNew">
   <property name="icon">
    <iconset resource="../../core/gui/core_rsc.qrc">
     <normaloff>:/core/icons/port.png</normaloff>:/core/icons/port.png</iconset>
   </property>
   <property name="text">
    <string>New Port...</string>
   </property>
  </action>
  <action name="actionPortEdit">
   <property name="text">
    <string>Edit Port...</string>
   </property>
  </action>
  <action name="actionDataViewImport">
   <property name="text">
    <string>Import Data View...</string>
   </property>
  </action>
  <action name="actionDataViewExport">
   <property name="text">
    <string>Export Data View...</string>
   </property>
  </action>
  <action name="actionDataViewEdit">
   <property name="text">
    <string>Edit Data View...</string>
   </property>
  </action>
  <action name="actionToolsSettings">
   <property name="text">
    <string>Settings...</string>
   </property>
  </action>
  <action name="actionPortDelete">
   <property name="text">
    <string>Delete Port</string>
   </property>
  </action>
  <action name="actionToolsSendMessage">
   <property name="icon">
    <iconset resource="client_rsc.qrc">
     <normaloff>:/client/icons/sendmessage.png</normaloff>:/client/icons/sendmessage.png</iconset>
   </property>
   <property name="text">
    <string>Send Message ...</string>
   </property>
  </action>
  <action name="actionDataViewItemNew">
   <property name="icon">
    <iconset resource="../../core/gui/core_rsc.qrc">
     <normaloff>:/core/icons/dataviewitem.png</normaloff>:/core/icons/dataviewitem.png</iconset>
   </property>
   <property name="text">
    <string>New Item(s)...</string>
   </property>
  </action>
  <action name="actionDataViewItemEdit">
   <property name="text">
    <string>Edit Item(s)...</string>
   </property>
  </action>
  <action name="actionDataViewItemInsert">
   <property name="text">
    <string>Insert Item</string>
   </property>
  </action>
  <action name="actionDataViewItemDelete">
   <property name="text">
    <string>Delete Item(s)</string>
   </property>
  </action>
  <action name="actionWindowDataViewShowAll">
   <property name="text">
    <string>Show All</string>
   </property>
  </action>
  <action name="actionWindowShowActive">
   <property name="text">
    <string>Show Active</string>
   </property>
  </action>
  <action name="actionWindowDataViewCloseAll">
   <property name="text">
    <string>Close All</string>
   </property>
  </action>
  <action name="actionWindowCloseActive">
   <property name="text">
    <string>Close Active</string>
   </property>
  </action>
  <action name="actionWindowCascade">
   <property name="text">
    <string>Cascade</string>
   </property>
  </action>
  <action name="actionWindowTile">
   <property name="text">
    <string>Tile</string>
   </property>
  </action>
  <action name="actionPortNewDevice">
   <property name="text">
    <string>New Device...</string>
   </property>
  </action>
  <action name="actionPortClearAllDevices">
   <property name="text">
    <string>Clear All Devices</string>
   </property>
  </action>
  <action name="actionPortImport">
   <property name="text">
    <string>Import Port...</string>
   </property>
  </action>
  <action name="actionPortExport">
   <property name="text">
    <string>Export Port...</string>
   </property>
  </action>
  <action name="actionDeviceImport">
   <property name="text">
    <string>Import Device...</string>
   </property>
  </action>
  <action name="actionDeviceExport">
   <property name="text">
    <string>Export Device...</string>
   </property>
  </action>
  <action name="actionDataViewImportItems">
   <property name="text">
    <string>Import Items...</string>
   </property>
  </action>
  <action name="actionDataViewExportItems">
   <property name="text">
    <string>Export Items...</string>
   </property>
  </action>
  <action name="actionToolsScanner">
   <property name="icon">
    <iconset resource="client_rsc.qrc">
     <normaloff>:/client/icons/scanner.png</normaloff>:/client/icons/scanner.png</iconset>
   </property>
   <property name="text">
    <string>Scanner...</string>
   </property>
  </action>
  <action name="actionEditEdit">
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionFileInfo">
   <property name="text">
    <string>Info...</string>
   </property>
  </action>
  <action name="actionFileRecent">
   <property name="text">
    <string>Recent</string>
   </property>
  </action>
  <action name="actionFileClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionWindowViewSubWindow">
   <property name="text">
    <string>View SubWindow</string>
   </property>
  </action>
  <action name="actionWindowViewTabbed">
   <property name="text">
    <string>View Tabbed</string>
   </property>
  </action>
  <action name="actionWindowCloseAll">
   <property name="text">
    <string>Close All</string>
   </property>
  </action>
  <action name="actionFileImportProject">
   <property name="text">
    <string>Import Project ...</string>
   </property>
  </action>
  <action name="actionToolsSendBytes">
   <property name="icon">
    <iconset resource="client_rsc.qrc">
     <normaloff>:/client/icons/sendbytes.png</normaloff>:/client/icons/sendbytes.png</iconset>
   </property>
   <property name="text">
    <string>Send Bytes ...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="client_rsc.qrc"/>
  <include location="../../core/gui/core_rsc.qrc"/>
 </resources>
 <connections/>
</ui>
